test/react_test_utils_test.dart

```diff
@@ -91,58 +97,101 @@ testUtils({isComponent2: false, dynamic eventComponent, dynamic sampleComponent,
         event(domNode, eventData);
         expect(domNode.text, equals('$eventName $fakeTimeStamp'));
       });
+
+      if (expectEventType != null) {
+        test('with correct type', () {
+          SyntheticEvent capturedEvent;
+          final component = renderIntoDocument(div({
+            eventHandlerName: (e) => capturedEvent = e,
+          }));
+          event(react_dom.findDOMNode(component), eventData);
```

Comment:
Should we use a ref on the div here instead since `findDOMNode` usage is a no-no these days?